themes/gatsby-theme-service-relief/gatsby-config.js

```diff
@@ -2,11 +2,13 @@ require("dotenv").config({
   path: `.env.${process.env.NODE_ENV}`
 });
 
-module.exports = ({ authorName, authorLink }) => {
+module.exports = ({ authorName, authorLink, description }) => {
   return {
     siteMetadata: {
       title: `${process.env.CITY} Service Relief`,
-      description: `A list of local service-industry businesses and their fundraisers to help them get through the local shutdowns.`,
+      description:
+        description ||
```

Comment:
I think siteMetadata is overwritable in the starter's siteMetadata by default? In other words, the algorithm is largely:

- take theme's siteMetadata
- Deeply extend with starters
- Starters values win out (if they exist)

Reply:
Oh, you're right! I knew you were smart. This already works without the change.